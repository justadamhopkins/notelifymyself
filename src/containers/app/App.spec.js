import React from 'react'
import { shallow } from 'enzyme'
import { App } from '.'

describe('<App/>', () => {
  it('Should render component', () => {
    const wrapper = shallow(<App/>)
    expect(wrapper).toMatchSnapshot()
  })
})
