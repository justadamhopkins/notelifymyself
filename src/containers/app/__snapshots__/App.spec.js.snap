// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> Should render component 1`] = `
<div>
  <Connect(Layout)>
    <Router>
      <Connect(Form)
        path="/category/:categoryId"
      />
      <Connect(Cards)
        path="/category/:categoryId"
      />
    </Router>
  </Connect(Layout)>
</div>
`;
